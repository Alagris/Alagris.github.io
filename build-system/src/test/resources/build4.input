:load sample.mealy
:eval function1 '01'
:eval function1 ''
:eval function3 'functions can be reused like thisthis function has no type, hence it defaults to using .* as alphabet'
:eval mirror 'reflect a input'
:eval mirror 'reflect b input'
:eval mirror 'reflect z input'
:load sample_load.mealy
:eval function1 'a'
//ignore this comment