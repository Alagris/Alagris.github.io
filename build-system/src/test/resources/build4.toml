projectName = "test project 1"

