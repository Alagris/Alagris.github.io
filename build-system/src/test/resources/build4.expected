'011'
'10'
''
'a'
'b'
'z'
'redefined function1'